scripts/coverage_report_jenkins.py

```diff
@@ -2,20 +2,44 @@
 # -*- coding: utf-8 -*-
 import argparse
 import jenkins
+import os
 import re
 import requests
 import subprocess
+import time
 
 from requests.auth import HTTPBasicAuth
 from urlparse import urlsplit, urlunsplit
 
 from cfme.utils.appliance import IPAppliance
+from cfme.utils.conf import env
```

Comment:
Comment not related to cfme.utils.conf.env, but to an import you would need for this suggestion.

Consider using cfme.utils.log.logger, and its method for logging to stdout as well as the cfme.log instead of all the `print` statements that you have here.

This is done in the utility scripts that Jenkins runs, here's one example:
* Import: https://github.com/ManageIQ/integration_tests/blob/master/scripts/cleanup_old_vms.py#L14
* Setup at module level: https://github.com/ManageIQ/integration_tests/blob/master/scripts/cleanup_old_vms.py#L29
* logger calls: https://github.com/ManageIQ/integration_tests/blob/master/scripts/cleanup_old_vms.py#L124

Note logger doesn't use the formatter, but passes variables for interpolation into the message: 
`print('string: {}'.format(var))` versus `logger.info('string: %s', var)`

Reply:
Will definitely do that.   This is inherited but I would prefer the logging be standard.